File: src/com/trapps/jvsk/java/CastingExample.java
package com.trapps.jvsk.java;

/**
 * @author Akash Patra
 */
public class CastingExample {

    /**
     * Check String type and then find String length
     *
     * @param name
     */
    public void stringCast(Object name) {
        if (name instanceof String) {
            int length = ((String) name).length();
            System.out.println("It is a String and of length: " + length);
        }
    }
}

File: src/in/co/trapps/jvsk/kotlin/CastingExample.kt
package `in`.co.trapps.jvsk.kotlin

/**
 * @author Akash Patra
 */
class CastingExample {

    /**
     * Check String type and then find String length
     *
     * @param name
     */
    fun stringCast(name: Any) {
        if (name is String) {
            val length = name.length
            println("It is a String of length: $length")
        }
    }
}